docs/basics/basic_benchmark.md

```diff
@@ -0,0 +1,139 @@
+# Basic Benchmark Structure
+
+
+Every benchmark starts with these global configuration.
+
+```yaml
+name: BENCHMARK_NAME
+http:
+  host: URL
+  sharedConnections: CONNECTIONS
+phases: ...
+```
+
+* URL is the HTTP protocol and hostname of the Application Under Test (e.g. http://myapp.com)
+* Shared connections are physical HTTP (1.x or2) connection.
+
+Example: 
+
+```yaml
+name: My_App
+http:
+  host: http://myapp.com:9080
+  sharedConnections: 10
+phases: ...
+```
+
+## Multiple Core Support
+
+* By default, Hyperfoil only utlize single core for the execution for experimenting & debugging.  For production usage, add the `threads` that matches to the total CPU core numbers.
```

Comment:
typo: utilizes only

Reply:
yeah, I am still working on English (it was D when I was in high school..) plural is the hardest because it doesn't exist in Japanese...

Reply:
Wow, I didn't know :) I'll just comment/fix when I see a mistake, no bad feelings about that.